src/main/resources/com/google/api/codegen/go/doc.snip

```diff
@@ -15,5 +15,20 @@
     @end
     package {@view.fileHeader.localPackageName} // import "{@view.importPath}"
 
-    const gapicNameVersion = "gapic/0.1.0"
+    import (
+        "golang.org/x/net/context"
+        "google.golang.org/grpc/metadata"
+    )
+
+    const (
+        gapicName = "gapic"
+        gapicVersion = "20170208"
```

Comment:
I'm afraid that we're still going to forget to update this. Could you add a context method to generate this date string for now, so that it always reflects the GAPIC build date? I'm less worried about the risk of multiple release cut on the same date.